nixos/modules/services/desktop-managers/plasma6.nix

```diff
@@ -245,6 +245,7 @@ in {
     systemd.services."drkonqi-coredump-processor@".wantedBy = ["systemd-coredump@.service"];
 
     xdg.icons.enable = true;
+    xdg.icons.fallbackThemes = ["breeze" "Breeze_Light"];
```

Comment:
```suggestion
    xdg.icons.fallbackThemes = ["breeze_cursors"];
```

Reply:
Applied.

Reply:
What package are these defined `breeze_cursors`? In `kdePackages.breeze-icons`, I only see `breeze` and `breeze-dark` directories.

Reply:
`kdePackages.breeze`